### PR TITLE
fix: Auth Provider

### PR DESCRIPTION
**Pull request by amitojsingh**
1. There were two Auth Providers which were conflicting with each other. I changed it to one.

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import React, { Suspense } from 'react';
 import i18n from 'i18next';
 import { initReactI18next, useTranslation } from 'react-i18next';
 import CONSTANTS from './constants';
-import { UserAuthContextProvider } from './auth';
+// import { UserAuthContextProvider } from './auth';
```
Comment (MySmilingTurban) [inline src/App.tsx:5]: nit: remove comment

File: src/auth/index.tsx [hunk 2]
```diff
@@ -23,9 +23,9 @@ import {
 import { firebaseErrorCodes as errors } from 'constants/errors';
 import roles from 'constants/roles';
 
-const UserAuthContext = createContext<any>(null);
+const AuthContext = createContext<any>(null);
```
Comment (MySmilingTurban) [inline src/auth/index.tsx:26]: can we change any to some relevant type?
maybe <Context | null>

Comment (MySmilingTurban) [reply src/auth/index.tsx]: @amitojsingh veerji, I added a new type to be able to remove any
would love your input on it to handle it better

**Review by MySmilingTurban — APPROVED** [review]
Looks good to me!

**Comment by MySmilingTurban** [comment]
@amitojsingh veerji, the component to save profile picture is not working on profile page.
I'll spent some time in the evening to fix that

**Comment by MySmilingTurban** [comment]
@amitojsingh veerji
Only one outstanding issue is left in this PR which is the component to save profile picture is not working on profile page.
As per my understanding, it is saving data in Firebase User and users collection, but after updation, the local data of user is not updated.

**Comment by amitojsingh** [comment]
Okay Veer ji shall we create a separate issue for this? @MySmilingTurban

**Pull request merged**